.gitignore

```diff
@@ -0,0 +1,2 @@
+myapp/bin/webpack
+myapp/bin/webpack-dev-server
```

Comment:
Why do you want to exclude these commands from Git?

Reply:
Where are these files displayed? Why is `myapp/bin/rails` allowed to be displayed somewhere while `webpack*` are not allowed?

Reply:
Running the server again updated my webpacker version so I think I should include it to be shown. I thought as it was already in my bin files I could just ignore it form displaying again. 👀

Reply:
OK. No problem